Header header

uint16 length
string data
